documentation/advanced-integration.md

```diff
@@ -153,7 +153,7 @@ window.StudioUI.studioLoaderConfig({
     projectName: projectName,
 
     /* outputTypes: object of all available output types (optional) that have a boolean value */
-    outputTypes: outputTypes,
+    outputSettings: outputSettings,
```

Comment:
I think we could and should deprecate this, it hasn't any value anymore (output is not possible without output setting)